.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thanks for creating an issue! Please fill out this form so we can be
+sure to have all the information we need, and to minimize back and forth.
+
+* **What are you trying to do?**
+
+* **What feature or behavior is this required for?**
+
+* **How could we solve this issue? (Not knowing is okay!)**
+
+* **Anything else?**
+
+cc @DarthHater @bhamail
```

Comment:
Q: Are these CCs intentional?  Couldn't they just turn on email?

Reply:
These CCs are definitely intentional. It bubbles up the PRs in our email so we get sent direct emails about them. There's likely a better way to do this but it was pretty quick to do this and required not learning anything about email filters (lol)